src/modules/hyprland/workspace.cpp

```diff
@@ -182,6 +182,11 @@ void Workspace::update(const std::string &format, const std::string &icon) {
     m_button.hide();
     return;
   }
+  if (this->m_workspaceManager.specialVisibleOnly() && \
+      this->isSpecial() && ! this->isVisible()){
```

Comment:
Sorry, suck at these code suggestion blocks... was just pointing out which file clang-format was complaining about.

```suggestion
  if (this->m_workspaceManager.specialVisibleOnly() && this->isSpecial() && !this->isVisible()) {
```